website/src/app/[lang]/[region]/(website)/campaign/[campaign]/page.tsx

```diff
@@ -47,127 +65,177 @@ export default async function Page({ params }: CampaignPageProps) {
+							</div>
+							<div className="my-8">
+								<Typography size="2xl" color="primary">
+									{campaign.description}
 								</Typography>
 							</div>
-						)}
-						{percentageCollected !== undefined && (
-							<div className={'mb-6'}>
-								<div className=" mb-1 flex font-medium">
-									<div className="text-accent w-1/2">
+							<div className="flex">
+								<div className="flex-1 text-left">
+									<Typography size="md" color="primary">
 										{translator.t('campaign.with-goal.collected-percentage', {
 											context: {
 												percentage: percentageCollected,
 											},
 										})}
-									</div>
-									<div className="text-primary w-1/2 text-right">{translator.t('campaign.with-goal.goal-title')}</div>
+									</Typography>
 								</div>
-								<div className="w-full">
-									<Progress value={percentageCollected} className={'h-5 '} />
+								<div className="flex-1 text-right">
+									<Typography size="md" color="primary">
+										{translator.t('campaign.with-goal.goal-title')}
+									</Typography>
 								</div>
-								<div className=" mt-1 flex font-medium">
-									<div className="text-accent w-1/2">
+							</div>
+							<div>
+								<Progress value={percentageCollected} className={'h-6'} />
+							</div>
+							<div className="flex">
+								<div className="flex-1 text-left">
+									<Typography size="md" color="primary">
 										{translator.t('campaign.with-goal.collected-amount', {
 											context: {
 												count: contributions,
 												amount: amountCollected,
 												currency: campaign.goal_currency,
 											},
 										})}
-									</div>
-									<div className=" text-primary w-1/2 text-right">
+									</Typography>
+								</div>
+								<div className="flex-1 text-right">
+									<Typography size="md" color="primary">
 										{translator.t('campaign.with-goal.goal-amount', {
 											context: {
 												amount: campaign.goal,
 												currency: campaign.goal_currency,
 											},
 										})}
-									</div>
+									</Typography>
 								</div>
 							</div>
-						)}
-					</div>
-					{daysLeft >= 0 && (
-						<>
-							<div className="flex flex-col items-center">
-								<Typography size="xl" weight="bold" color="accent">
-									{translator?.t('campaign.days-left', { context: { count: daysLeft } })}
-								</Typography>
-							</div>
-							<div className="mt-4 w-full">
-								<OneTimeDonationForm
-									lang={params.lang}
-									region={params.region}
-									translations={{
-										oneTime: translator.t('donation-interval.0.title'),
-										monthly: translator.t('donation-interval.1.title'),
-										amount: translator.t('amount'),
-										submit: translator.t('button-text'),
-									}}
-									campaignId={params.campaign}
-								/>
+						</div>
+						<div className="flex items-center justify-center" style={{ height: '500px' }}>
```

Comment:
Shall we keep the `{daysLeft >= 0 && (` so that the donation form is not shown any more after the deadline?
Or what is the purpose of the end date?